src/index.html

```diff
@@ -4,9 +4,206 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo_link" href="#">
+        <img src="./images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="product">
+      <div class="product__list">
+        <div class="product__item" data-qa="card">
+          <img class="product__photo" src="./images/imac.jpeg" alt="apple_a1419_imac">
+          <h3 class="product__name">
+            <a class="product__link" href="#">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</a>
+          </h3>
+          <p class="product__code">Код товара: 195434</p>
+          <div class="product__reviews">
+            <div class="product__stars">
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+            </div>
+            <a href="#" class="product__review--link">Отзывов: 5</a>
```

Comment:
це швидше просто елемент, а не модифікатор
```suggestion
            <a href="#" class="product__review-link">Отзывов: 5</a>
```